libraries/api-extractor-model/src/mixins/ApiItemContainerMixin.ts

```diff
@@ -134,7 +137,7 @@ export function ApiItemContainerMixin<TBaseClass extends IApiItemConstructor>(
       jsonObject: IApiItemContainerJson
     ): void {
       baseClass.onDeserializeInto(options, context, jsonObject);
-
+      options.shouldSortMembers = false;
```

Comment:
New edge case I thought of. What if we deserialize an `.api.json` file with sorted enum members, then add a new enum member via `ApiEnum.addMember`? Given `shouldSortMembers` is false, then subsequent calls to `members` won't be sorted. And we have no way of knowing whether the enum members were sorted, because that info was not serialized in the first place. 🤔  @octogonz, I think we might need to serialize the setting (per our last [Zulip discussion](https://rushstack.zulipchat.com/#narrow/stream/262521-api-extractor/topic/Adding.20project.20config.20for.20preserving.20Enum.20member.20order/near/283011274)).

Reply:
Discussed with @octogonz, we concluded that if someone wanted to modify an API doc model post initial serialization via something like `ApiEnum.addMember`, onus is on them to perform any necessary sorting. Thus, we'll keep this PR as-is and not serialize the setting.

Reply:
Sounds good, will resolve conversation